rx888_source/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 cmake_minimum_required(VERSION 3.13)
 project(rx888_source)
 
+if (MSVC)
+  set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
```

Comment:
This is not needed for modules and is not recommended at all in general. Only use for the core software since it's needed for modules to bind to it.